nemo/collections/asr/losses/rnnt.py

```diff
@@ -60,9 +158,11 @@ def output_types(self):
         """
         return {"loss": NeuralType(elements_type=LossType())}
 
-    def __init__(self, num_classes, reduction='mean_batch'):
+    def __init__(self, num_classes, reduction: str = 'mean_batch', loss_name: str = "default", loss_kwargs=None):
```

Comment:
Where reduction is used?

Reply:
Ah good catch. The reduction in this constructor is the main one. The override in loss_kwargs should practically not be used. Ill fix that.

Reply:
Removed the reduction option inside the resolver function, and now these two classes dont take any extra parameters. Im thinking of future case where loss resolver may optionally require some more arguments (while these two dont require any).